optuna/structs.py

```diff
@@ -95,7 +100,7 @@ class FrozenTrial(
             :class:`~optuna.study.Study.study_id` to identify a :class:`~optuna.trial.Trial`.
     """
 
-    internal_fields = ['params_in_internal_repr', 'trial_id']
+    internal_fields = ['distributions', 'params_in_internal_repr', 'trial_id']
```

Comment:
I added `distributions` to the list, because I think that to include distribution information to pandas dataframes may be noisy for ordinal users.